test/types/checkbox.test.ts

```diff
@@ -0,0 +1,43 @@
+import { Tester } from '@mighty-justice/tester';
+
+import { TYPE_GENERATORS } from '../factories';
+
```

Comment:
Is the space between these imports standard?

Reply:
Yeah, we usually put blank spaces between:
```
global imports
ant-design imports
@mighty-justice imports
../ imports
./ imports
style.module.less import
```